### DISCUSSION TITLE
allow templates for values in tile card

### DISCUSSION THREAD

**Discussion by ChristophCaina** [home-assistant/frontend#21256] [Dashboard]
If you want to modify the information or how a value will be shown, you need either a complete template sensor, or - you can try to use some card-mod hacks to modify the value information (secondary, for example)

I am doing this here, as an example:

```
type: tile
entity: sensor.lxc_samba_server_100_betriebszeit
name: letzter Start
layout_options:
  grid_columns: 4
  grid_rows: 1
vertical: false
hide_state: false
card_mod:
  style:
    ha-tile-info$: |
      .secondary {
         visibility: hidden;
       }
       .secondary:before {
         visibility: visible;
         content: "{{ as_timestamp(states(config.entity) | title) | timestamp_custom("%d.%m.%Y - %H:%M:%S") }}"
       }
 ```
 This card mod will change the current value into a date / time information.
 The Tile-Card looks like this:
 ![grafik](https://github.com/home-assistant/frontend/assets/26391061/137616e7-4172-42dc-962e-8a95014ac546)

Without card mod, it is showing the following value:
![grafik](https://github.com/home-assistant/frontend/assets/26391061/80414878-2b58-452e-a423-d47b6519b89c)

It would be really great, if you could have an option, to apply templates directly to the card, without the need of card-mod or another templated sensor.
this would allow some flexibility - by default, everything would work as known... the template option could also be just available in yaml, to be accessible for users with more experience.

**Comment by BBE-FR** [comment]
Fully support an advanced tile with templating.

I made an even more complete proposal here: https://github.com/home-assistant/frontend/discussions/19930
